atlp-rwanda/e-commerce-lydia-32-fn
# Ft test coverage#187570807

### What does this PR do?
This Pull Request serves to setup testing environment for our components
### Description of Task to be completed?
1. Setting Jest testing environment.
2. Running some sample tests to see if they can work
### How should this be manually tested?
Inside root directory run **npm run test** and **npm run test:coverage** if you want to see coverage
